src/pages/Categories/CategoriesBeta.vue

```diff
@@ -136,6 +135,19 @@ export default {
 		KvButton,
 		LoanSpotlight
 	},
+	metaInfo() {
+		return {
+			title: 'Choose a category and fund a loan | Kiva',
```

Comment:
You don't need the `| Kiva` part, sub routes like this get their title inserted into the template in: `src/App.vue:27`